stravalib/model.py

```diff
@@ -459,8 +714,9 @@ class ActivityPhoto(BackwardCompatibilityMixin, DeprecatedSerializableMixin):
     created_at: Optional[datetime] = None
     created_at_local: Optional[datetime] = None
     location: Optional[LatLon] = None
-    urls: Optional[Dict] = None
-    sizes: Optional[Dict] = None
+    urls: Optional[Dict[str, str]] = None
+    # TODO - is this a float return?
```

Comment:
There's still a TODO here I'm not sure what you meant

Reply:
ah yes - for sizes i assumed that this relates to photo sizes and would thus be of type float. BUT could it be an int or some other numeric value? in short, I wasn't sure how to be certain that float  was the best type here. 

because this is an undocumented entity i feel like i'm guessing a bit which doesn't feel great.